src/Assets/LightSkyDefense/Scripts/Waves/CycleWaves.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "New CycleWaves", menuName = "New CycleWaves", order = 1)]
```

Comment:
Don't have "New" in the name

Reply:
More of an editor thing than structure thing.